src/Audit.php

```diff
@@ -285,7 +299,15 @@ public function getModified(bool $json = false, int $options = 0, int $depth = 5
             }
         }
 
-        return $json ? json_encode($modified, $options, $depth) : $modified;
+        if ($json === true) {
+            $modified = json_encode($modified, $options, $depth);
+
+            if ($modified === false) {
```

Comment:
Here this code is repeated twice, why not just elvis ternary and early return
Something like this
```php
    // it seems more readable to me
    if (! $json) {
        return $data;
    }

    return json_encode($data, $options, $depth) ?: '{}';
```
Also, If it should be a json it should not be `return '{}';` instead of `return [];`
let's avoid if inside if, unnecessarily

Reply:
I'll give it a go and see what stan thinks :pray: